javascript-loops-practice/filter-out-nulls.js

```diff
@@ -0,0 +1,13 @@
+/* exported filterOutNulls */
+function filterOutNulls(values) {
+  var tempValues = values;
+  values = [];
```

Comment:
Instead of resaving your parameter into a new variable immediately, you should just create a new variable for your output:
`var output = []`
Then add to that in the loop, so you can use the original `values` parameter to loop over